README.md

```diff
@@ -1,2 +1,21 @@
-# stockanalyzer
+# Stockanalyzer
 An application which manages and aggregates stock data
+
+## Description
+
+Stockanalyzer uses external API to retrieve stock data and persists it to an in-memory database.
+The aggregation process happens every 1 minute.
+
+There are two feature now:
++ Retrieve all manageable stocks symbols from db
++ Find five best stocks to based on current database data
+
+Entities:
+```kotlin 
+  data class Stock (var id: UUID? = null, val symbol: String?, 
```

Comment:
- Please describe each field of Stock. What does it mean. Describe your domain more detailed.
- Add User model if you want to implement subscriptions and notifications

Reply:
User will be implemented later